config/install/core.entity_form_display.node.publication_landing_page.default.yml

```diff
@@ -4,6 +4,7 @@ dependencies:
   config:
     - field.field.node.publication_landing_page.body
     - field.field.node.publication_landing_page.field_localgov_services_landing
+    - field.field.node.publication_landing_page.field_publication
```

Comment:
Sorry to be a pedant, but I think we probably want to follow the localgov_ field prefix convention for all our fields.